pkg/ui/react-app/src/thanos/Navbar.tsx

```diff
@@ -104,6 +104,7 @@ const Navigation: FC<PathPrefixProps & NavigationProps> = ({ pathPrefix, thanosC
     <Navbar className="mb-3" dark color="dark" expand="md" fixed="top">
       <NavbarToggler onClick={toggle} className="mr-2" />
       <Link className="navbar-brand" to={`${pathPrefix}${defaultRoute}`}>
+        <img src="https://thanos.io/icon-dark.png" alt="thanos" style={{ width: '2rem', marginRight: '0.5rem' }} />
```

Comment:
I believe it would be interesting to have it locally. Otherwise server that doesn't have access to internet won't be able to display the logo :/

Reply:
In case it can help, this is how we did it in Prometheus : https://github.com/prometheus/prometheus/pull/10236/files

Reply:
Hey @Nexucis looks alright?

Reply:
could you run the command `make assets` at the root of the project ? And then commit the modification of the files

Reply:
I already did that, there isn't any modification apart from what I committed

Reply:
after running `make assets` : 

```bash
git status
On branch logo
Your branch is up to date with 'origin/logo'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   pkg/ui/bindata.go

no changes added to commit (use "git add" and/or "git commit -a")
```

Reply:
maybe you didn't push every changes

Reply:
excepting the asset issue, it looks good to me. I'm just wondering if it would be cleaner to put the style in a dedicated css class.
@onprem any thought around this ?

Reply:
> maybe you didn't push every changes

Do you want me to add the bindata.go file as well, it had a lot of changes that's why I didn't add it!

Reply:
yes that's exactly what you have to do